migrations/sql/V2019.12.21.23.39__create_mine_incident_category_codes_table.sql

```diff
@@ -0,0 +1,25 @@
+ALTER TABLE mine_incident DROP CONSTRAINT IF EXISTS mine_incident_mine_incident_category_code_fkey;
+ALTER TABLE mine_incident DROP COLUMN IF EXISTS mine_incident_category_code;
+
+ALTER TABLE mine_incident_category ALTER COLUMN mine_incident_category_code TYPE varchar(3);
```

Comment:
I changed mine_incident_category_code across the board to restrict its size to 3 characters.